en/docs/reference/synapse-properties/endpoint-properties.md

```diff
@@ -542,7 +542,7 @@ The following is an HTTP endpoint configured with basic authentication.
 The following properties <b>only</b> apply to HTTP endpoint. 
 
 !!! Note
-      1. You can also use environment variables for these parameters. For more information, see [Externalized Configuration]({{base_path}}/develop/externalized-configuration.md).
+      1. You can also use environment variables for these parameters. For more information, see [Externalized Configuration]({{base_path}}/develop/externalized-configuration/).
```

Comment:
```suggestion
      1. You can also use environment variables for these parameters. For more information, see [Externalized Configuration]({{base_path}}/develop/externalized-configuration).
```

Reply:
MkDocs redirect `{{base_path}}/develop/externalized-configuration` to `{{base_path}}/develop/externalized-configuration/`. Hence removing the ending `/` will have an additional redirect call.